compiler/rustc_smir/src/stable_mir/ty.rs

```diff
@@ -0,0 +1,571 @@
+use rustc_middle::mir::interpret::{alloc_range, AllocRange, ConstValue, Pointer};
```

Comment:
cc @rust-lang/project-stable-mir

Reply:
was this added here by mistake? This seems to be a copy of `stable_mir/src/ty.rs`